src/Storage/Storage.Autorest/test/Start-AzStorageAccountMigration.Tests.ps1

```diff
@@ -0,0 +1,41 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Start-AzStorageAccountMigration'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'Start-AzStorageAccountMigration.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'Start-AzStorageAccountMigration' {
+    It 'CustomerExpanded' -skip {
+        { throw [System.NotImplementedException] } | Should -Not -Throw
+    }
+
+    It 'CustomerViaJsonString' -skip {
+        { throw [System.NotImplementedException] } | Should -Not -Throw
+    }
+
+    It 'CustomerViaJsonFilePath' -skip {
+        { throw [System.NotImplementedException] } | Should -Not -Throw
+    }
+
+    It 'Customer' -skip {
+        { throw [System.NotImplementedException] } | Should -Not -Throw
+    }
+
+    It 'CustomerViaIdentityExpanded' -skip {
+        { throw [System.NotImplementedException] } | Should -Not -Throw
+    }
+
+    It 'CustomerViaIdentity' -skip {
+        { throw [System.NotImplementedException] } | Should -Not -Throw
+    }
```

Comment:
Please remove the useless test cases for the removed parameter set.
Add necessary test cases.

Reply:
Done. 
As discussed offline, the cmdlet takes at least 5 days to finish, so it's hard to add tests to record here. Will test locally.